index.js

```diff
@@ -14,4 +14,150 @@ menuItem.forEach((menuItem) => {
   menuItem.addEventListener('click', () => {
     nav.style.display = 'none';
   });
+});
+
+// PROJECT POPUP
+const projects = [
+  {
+    name: 'Tonic',
+    description: 'Lorem Ipsum is simply dummy text of the prinhe 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.',
+    image: './img/Dbg1.png',
+    stack: ['FACEBOOK', 'Full Stack Dev', '2015'],
+    technologies: ['HTML', 'CSS', 'JavaScript', 'Ruby', 'Bootstrap'],
+    liveLink: '',
+    sourceLink: '',
+  },
+  {
+    name: 'Multi-Post Stories',
+    description: 'Lorem Ipsum is simply dummy text of the prinhe 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.',
+    image: './img/Dbg2.png',
+    stack: ['FACEBOOK', 'Full Stack Dev', '2015'],
+    technologies: ['HTML', 'CSS', 'JavaScript', 'Ruby', 'Bootstrap'],
+    liveLink: '',
+    sourceLink: '',
+  },
+  {
+    name: 'Facebook 360',
+    description: 'Lorem Ipsum is simply dummy text of the prinhe 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.',
+    image: './img/Dbg3.png',
+    stack: ['FACEBOOK', 'Full Stack Dev', '2015'],
+    technologies: ['HTML', 'CSS', 'JavaScript', 'Ruby', 'Bootstrap'],
+    liveLink: '',
+    sourceLink: '',
+  },
+  {
+    name: 'UBER Navigation',
+    description: 'Lorem Ipsum is simply dummy text of the prinhe 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.',
+    image: './img/Dbg4.png',
+    stack: ['FACEBOOK', 'Full Stack Dev', '2015'],
+    technologies: ['HTML', 'CSS', 'JavaScript', 'Ruby', 'Bootstrap'],
+    liveLink: '',
+    sourceLink: '',
+  },
+];
+
+const cardsContainer = document.querySelector('#About');
+
+projects.forEach((project) => {
+  const card = document.createElement('div');
+  card.classList.add('about-section');
+
+  card.innerHTML = `
+    <img src="${project.image}" class="card-img" />
+      <div class="canopy">
+        <h4 class="about-head">${project.name}</h4>
+        <ul class="cano">
+          <li class="tec">FACEBOOK</li>
+          <li class="tech">Full Stack Dev</li>
+          <li class="tech">2015</li>
+        </ul>
```

Comment:
- [ ] Well! You have manually typed the above list. However, as per the project requirements, you need to add them dynamically based on the project's array of objects. 🔄 Please consider updating your code to dynamically generate the themes based on the data of the project.

By implementing dynamic generation of themes, your code will be more scalable and maintainable, as it will adapt to changes in the project's array without manual intervention.💪🌟